webdriver/support/http_request.py

```diff
@@ -9,14 +10,19 @@ def __init__(self, host, port):
     def head(self, path):
         return self._request("HEAD", path)
 
-    def get(self, path):
-        return self._request("GET", path)
+    def get(self, path, use_json=False):
```

Comment:
these changes to allow for parsing of json only matter if we decide we want the sessionless status response, since in that case we don't have a wd client that is already doing the parsing.